src/components/Extension/DocuSignExtension/actions/EmbeddedSigningAction/hooks/useCompleteEmbeddedSigningAction.tsx

```diff
@@ -0,0 +1,20 @@
+import { useCallback } from 'react'
+import { DataPoints, useCompleteExtensionActivity } from '../../../types'
+
+export const useCompleteEmbeddedSigningAction = () => {
+  const { isSubmitting, onSubmit: _onSubmit } = useCompleteExtensionActivity()
+
+  const onSubmit = useCallback(
+    async (activityId: string) => {
+      const dataPoints: DataPoints = []
```

Comment:
Thinking about the current limitation we have to work with vis-a-vis the TTL of the signing link: wouldn't it be a good idea to return a `status` or `signed` data point that reflects the actual status of the document?

**Brainstorm:**
- A data point called `signed` (boolean)
- When the document is actually signed the boolean is set to `true`
- In all other cases it is set to `false` (eg: when TTL expired but the user completes the activity to move to the next one)

At least then we have some way of knowing whether the document was actually signed or not. Additionally, this would allow our users to build conditional logic in their care flow to repeat the process.

If you agree, I would also do this for the DropboxSign extension (separate MR).

Reply:
Sure, I can do it. But before I'll do it, I need to add a change in awell-extensions to create new data points for those 2 extensions and actions (docusign and dropboxsign)